openapi/openapi.yaml

```diff
@@ -1017,6 +1025,13 @@ components:
               type: object
               description: |
                 a place to store arbitrary system details you want to remember
+            system_tags:
```

Comment:
I feel like we may want to include more fields for tags in the future, that's why I use object instead of plain string.

- Good idea. Changed.
- I call it `system_tags` to match other fields, e.g., `system_info`, `system_id`, `system_details`, etc.

Reply:
Thanks for clarifying! Do you have an example of a possible use case that requires more fields for tags? I kinda think this may not be necessary. If we don't have a use case in mind, this may end up not being used but it makes the system more complex.

Reply:
Good question... I think we need a separate database to store all tags (you also suggested this). Each tag may also have fields like `tag_id`, `count` and `used_by` (a list of system ids). 

Some example use cases that I can think of right now: 
- A user wants to see all `LSTM` models, we can simply read from `used_by` instead of going through all systems.
- A user mis-spells a tag and deletes it immediatly. By removing it when `count == 0`, we can make tag suggestions more useful.

Reply:
Correct me if I am wrong, but I don't think we need a separate database/collection for these two fields. If we want to find all systems that have the `LSTM` tag, we can just filter on the `systems` collection. This would be fast if we have an index on the `system_tags` field. Similarly, this is also true for the count.

references:
count: https://www.mongodb.com/docs/manual/reference/method/db.collection.count/
multikey indexes: https://www.mongodb.com/docs/manual/core/index-multikey/#:~:text=To%20index%20a%20field%20that,%2C%20numbers)%20and%20nested%20documents.

Reply:
Thanks for sending the resources! I didn't know that. Yes, it makes sense now. I think I need to read more about NoSQLs. They seem powerful.

For the DB change, how about we do it in another PR? For this PR, we only attach a few strings to each submitted system and display them in the table. We can use this issue to keep track of the steps #357